packages/react/src/themes/base/components/Checkbox/checkboxVariables.ts

```diff
@@ -1,79 +1,75 @@
 import { pxToRem } from '../../../../lib'
 
 export type CheckboxVariables = {
-  checkboxBackground: string
-  checkboxBorderColor: string
-  checkboxBorderStyle: string
-  checkboxBorderRadius: string
-  checkboxBorderWidth: string
+  background: string
+  borderColor: string
+  borderStyle: string
+  borderRadius: string
+  borderWidth: string
   checkboxColor: string
-  checkboxGap: string
-  checkboxMargin: string
-  checkboxPadding: string
+  gap: string
+  margin: string
+  padding: string
 
   toggleBackground: string
   toggleBorderColor: string
   toggleBorderStyle: string
   toggleBorderRadius: string
   toggleBorderWidth: string
-  toggleColor: string
+  toggleIndicatorColor: string
   toggleMargin: string
   togglePadding: string
 
-  checkedCheckboxBackground: string
-  checkedCheckboxBorderColor: string
-  checkedCheckboxColor: string
-  checkedToggleColor: string
-  checkedToggleBackground: string
-  checkedToggleBorderColor: string
-  checkedTogglePadding: string
+  checkedBackground: string
+  checkedBorderColor: string
+  checkboxCheckedColor: string
+  checkboxToggleCheckedColor: string
```

Comment:
I think that `checkedToggleColor`, `checkedToggleBackground` and `checkedToggleBorderColor` namings were fine.